tests/src/test_cases/draw/test_draw_svg.c

```diff
@@ -308,6 +308,25 @@ void test_draw_shapes(void)
     draw_svg(svg);
     draw_snapshot(SNAPSHOT_NAME(svg_shapes_11));
     lv_svg_node_delete(svg);
+
+    const char * svg_shapes_12 = \
+                                 "<svg width=600 height=200 viewBox=\"0 0 1200 400\" xmlns=\"http://www.w3.org/2000/svg\" version=\"1.1\">"
+                                 "<rect x=\"1\" y=\"1\" width=\"1198\" height=\"398\" fill=\"none\" stroke=\"blue\" stroke-width=\"1\"/>"
+                                 "<path d=\"M300,200 h-150 a150,150 0 1,0 150,-150 z\" fill=\"red\" stroke=\"blue\" stroke-width=\"5\"/>"
+                                 "<path d=\"M275,175 v-150 a150,150 0 0,0 -150,150 z\""
+                                 " fill=\"yellow\" stroke=\"blue\" stroke-width=\"5\"/>"
+                                 "<path d=\"M600,350 l 50,-25"
+                                 " a25,25 -30 0,1 50,-25 l 50,-25"
+                                 " a25,50 -30 0,1 50,-25 l 50,-25"
+                                 " a25,75 -30 0,1 50,-25 l 50,-25"
+                                 " a25,100 -30 0,1 50,-25 l 50,-25\""
+                                 " fill=\"none\" stroke=\"red\" stroke-width=\"5\"/></svg>";
+
+    svg = lv_svg_load_data(svg_shapes_12, lv_strlen(svg_shapes_12));
+    TEST_ASSERT_NOT_EQUAL(NULL, svg);
```

Comment:
```suggestion
    TEST_ASSERT_NOT_NULL(svg);
```

Fyi, for future reference :)